assets/opstools/AppBuilder/classes/platform/views/ABViewDataview.js

```diff
@@ -142,11 +142,13 @@ module.exports = class ABViewDataview extends ABViewDataviewCore {
             datacollection: dc
          });
 
-         if (dc.datacollectionLink && dc.fieldLink) {
-            dc.bind(dataView, dc.datacollectionLink, dc.fieldLink);
-         } else {
-            dc.bind(dataView);
-         }
+         // NOTE: 'flexlayout' does not apply webix.AtomDataLoader
+         //
+         // if (dc.datacollectionLink && dc.fieldLink) {
+         //    dc.bind(dataView, dc.datacollectionLink, dc.fieldLink);
+         // } else {
+         //    dc.bind(dataView);
+         // }
```

Comment:
Ahh...this explains why we needed all of the individual listeners. I think we need a "create" or something similar listener to round out the features.

Reply:
I think I need this code here because it helps me later identify any UI that may be stale and needs to be refreshed when the data collection is updated.

https://github.com/appdevdesigns/appbuilder_class_core/blob/b84d52d33dc21e1bfe360974b67d53ad167d2c58/ABDataCollectionCore.js#L778

Does having this `.bind()` code cause errors?

Reply:
The dataview widget is using [webix.flexlayout](https://docs.webix.com/api__refs__flexlayout.html).
It does not inherit  [webix.AtomDataLoader](https://docs.webix.com/api__refs__atomdataloader.html) 

If we call .bind function of ABDataCollection, then [flexlayout](https://docs.webix.com/api__refs__flexlayout.html) [bind to webix.DataCollection](https://github.com/appdevdesigns/app_builder/blob/develop/assets/opstools/AppBuilder/classes/platform/ABDataCollection.js#L313
)

It causes from [this error](https://github.com/digi-serve/planning/issues/394) https://github.com/digi-serve/planning/issues/394.

Reply:
I added a method to track flexlayouts.